WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -834,14 +842,14 @@ private extension EditableOrderViewModel {
         for product in products {
             // Only perform the operation if the product has not been already added to the existing Order
             if !itemsInOrder.contains(where: { $0.productID == product.productID }) {
-                productInputs.append(OrderSyncProductInput(product: .product(product), quantity: 1))
+                productInputs.append(OrderSyncProductInput(product: .product(product), quantity: 1, discount: 0))
```

Comment:
We'll be passing a different value as `discount` property in these cases where we initialize it as `0`? If that's not the case, then we could update the `OrderSyncProductInput` to provide a default value:
```diff
struct OrderSyncProductInput {
    enum ProductType {
        case product(Product)
        case productID(Int64)
        case variation(ProductVariation)
    }
    var id: Int64 = .zero
    let product: ProductType
    let quantity: Decimal
-    let discount: Decimal
+    var discount: Decimal = .zero

    func updating(id: Int64) -> OrderSyncProductInput {
        .init(id: id, product: self.product, quantity: self.quantity, discount: discount)
    }
}
```

Reply:
Good point! Done in aaa3a1d44cfb7b71280fe877ce99c57bf204fbdb